src/utilities/hooks.ts

```diff
@@ -0,0 +1,5 @@
+import { TypedUseSelectorHook, useSelector } from 'react-redux';
+
+import { RootState } from '@store/interfaces';
+
+export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector;
```

Comment:
Do you want us to use the `useDispatch` the way it is explained in the [Redux docs](https://react-redux.js.org/using-react-redux/usage-with-typescript#define-typed-hooks)?

Reply:
Let's do this in another PR. It requires a bit more effort than the docs are indicating and I want to ship the new store structure.